pytgcalls/pytgcalls/mtproto/pyrogram_bridge.py

```diff
@@ -105,21 +105,21 @@ async def check_group_call(self) -> bool:
 
         return in_group_call
 
-    async def get_my_volume(self):
-        if not self.full_chat or not self.full_chat.call or not self.my_peer or not self.my_ssrc:
+    async def get_group_participants(
+        self, participants: list = [], sources: list = [], offset: str = "", limit: int = 1
```

Comment:
List is mutable. You mustn’t set list as default value of arg. The biggest mistake of programmers in Python